sdk/containerapps/Azure.ResourceManager.AppContainers/src/Generated/Models/ContainerAppLogin.cs

```diff
@@ -93,7 +93,7 @@ public string RoutesLogoutEndpoint
         /// This is an advanced setting typically only needed by Windows Store application backends.
         /// Note that URLs within the current domain are always implicitly allowed.
         /// </summary>
-        public IList<string> AllowedExternalRedirectUrls { get; }
+        public IList<Uri> AllowedExternalRedirectUrls { get; }
```

Comment:
since we can't simply change the return type, it would be better to add config in `autorest.md` to revert this change.